### PR TITLE
KingHost added

### PR DESCRIPTION
**Pull request by angelitomg**

### CODE REVIEW

File: _data/hosts.yml [hunk 1]
```diff
@@ -802,6 +802,38 @@
             version: 5.5.1
             semver: 5.5.1
 -
+    name: KingHost
+    url: 'https://kinghost.com.br/'
+    type: shared
+    default: 53
```
Comment (philsturgeon) [inline _data/hosts.yml]: Woah really?!

File: _data/hosts.yml [hunk 2]
```diff
@@ -802,6 +802,38 @@
...
+            phpinfo: null
+            patch: 18
+            version: 5.6.18
+            semver: 5.6.18
+        70:
+            phpinfo: null
+            patch: 3
+            version: 7.0.3
+            semver: 7.0.3
+-
```
Comment (philsturgeon) [inline _data/hosts.yml]: This double `-` will cause some trouble

Comment (angelitomg) [inline _data/hosts.yml]: Hi buddy, whats up? :) I tested this PHP versions on Kinghost today. All worlked well.

Thanks.

Cheers.

**Comment by philsturgeon** [comment]
Sadly the double - is breaking the build.

screen shot 2016-08-22 at 10 20 11 am

https://circleci.com/gh/philsturgeon/phpversions.info/489

**Comment by philsturgeon** [comment]
Fixed!

**Pull request merged**